apps/fluent-tester/src/RNTester/FabricTester.tsx

```diff
@@ -25,6 +25,83 @@ export interface IFabricTesterProps {
   enabledTests: TestDescription[];
 }
 
+const Header: React.FunctionComponent<{}> = () => {
+
+  const [selectedPlatform, setSelectedPlatform] = React.useState('win32');
+  const [selectedApp, setSelectedApp] = React.useState('office');
+  const [selectedTheme, setSelectedTheme] = React.useState('default');
+
+  const themeColor = () => {
+
+    var color = "black"; // default: office (black)
+
+    if (selectedApp == 'office') color = 'black';
+    if (selectedApp == 'word') color = "#2B579A";
+    if (selectedApp == 'excel') color = "#217346";
+    if (selectedApp == 'powerpoint') color = "#B7472A";
+    if (selectedApp == 'outlook') color = "#106EBE";
+
+    return { color: color };
+  }
+
+  const onAppChange = React.useCallback((appValue: string) => {
+    setSelectedApp(appValue);
+  }, []);
+
+  return (
+    <View style={fabricTesterStyles.header}>
+      <Text style={[fabricTesterStyles.testHeader, themeColor()]} testID={BASE_TESTPAGE}>
+        ⚛ FluentUI Tests
+      </Text>
+
+      <View style={fabricTesterStyles.pickerRoot}>
+        <View style={fabricTesterStyles.picker}>
+          <Text style={fabricTesterStyles.pickerLabel}>Platform:  </Text>
+          <Picker
+            selectedValue={selectedPlatform}
+            style={fabricTesterStyles.dropdown}
+            onValueChange={(platformValue) => setSelectedPlatform(platformValue)}
+          >
+            <Picker.Item label="Win32" value="win32" />
+            <Picker.Item label="UWP" value="uwp" />
+            <Picker.Item label="iOS" value="ios" />
+            <Picker.Item label="macOS" value="mac" />
+            <Picker.Item label="Android" value="android" />
```

Comment:
Is this changing the available tests to what is there for each platform? How would that work if a platform has native components?

Reply:
I believe this will allow us to import styling to make the components look like another platform, particularly for web where it will be ideal to preview ios/android/windows/win32/macos.

Reply:
^ What Jason said! Also, in terms of theme change, so far I was able to get header colors to change when you toggle the App category. Eventually, we will want to be able to toggle styling between the platform, app, and theme so I set up the dropdown mechanism for that but all of the options don't have styling attached to it yet.